server/blob/blob-objectstorage/pom.xml

```diff
@@ -140,6 +140,10 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>jcl-over-slf4j</artifactId>
```

Comment:
should be before the previous one?

Reply:
Well, slf4j api is still used in our code while the aws-3 lib requires the jcl-over-slf4j

Reply:
just talking about alphabetic order in the pom

Reply:
Ahhhh